editor/src/clj/editor/markdown.clj

```diff
@@ -433,18 +433,18 @@
 (g/defnode MarkdownNode
   (inherits r/CodeEditorResourceNode)
 
-  (output html g/Str :cached (g/fnk [save-value]
-                                    (str "<!DOCTYPE html>"
-                                         "<html><head></head><body>"
-                                         (-> save-value
-                                             data/lines->string
-                                             markdown->html)
-                                         "</body></html>"))))
+  (output html g/Str :cached (g/fnk [arg-list] [save-value]
```

Comment:
`[arg-list] [save-value]`? What's going on here?

Reply:
Me, trying to fix auto-indent in CSCode before committing the changes.. Sorry about that.